rollup.config.js

```diff
@@ -9,6 +9,20 @@ import { dependencies } from './package-lock.json'
 
 import fs from 'fs'
 import path from 'path'
+import write from 'write'
+
+import * as edge from 'edge.js'
+edge.registerViews(path.join(__dirname, './packages/shell-chrome/views'))
+const renderTemplate = () => {
+    let template = edge.render('master')
```

Comment:
can we call it `panel`? 😄 tiny bit more descriptive (though not much)

Reply:
We can indeed 😉 Will sort.